### ISSUE TITLE
"Linked" arrows should be arrows, not Lemmings

### ISSUE THREAD

**Issue by catb0t** [soscripted/sox#39]
**Installed Version: 1.0.3DEV  |  Environment: Tampermonkey/Chrome**

TL;DR the arrows in the Linked section of a question, borne of [here](http://meta.stackexchange.com/questions/276235), seem to all point "here" rather than reflect reality.

**Expected Behavior**

"Linked-from-here" arrows point NE, "away" from the question, while "Linked-to-here" arrows point SW, "towards" the question.

**Actual Behavior**

**All** the arrows for **all** "Linked" sections on **all** sites on the SE network point "towards" the question regardless of where the link is actually. The tooltips, however, are correct, it's just the arrow icons themselves that **all** point the same wrong direction... [kinda like lemmings](https://en.wikipedia.org/wiki/Lemmings_(video_game)) :D.

**Error Logs**

Nope. But you can have an example that should definitely point "away" from the question:

* http://meta.stackexchange.com/q/276235/ -- the singular entry is linked *by* this question, not vice-versa.

**Steps to reproduce**

Load a question that links or is linked to by another question on the same site.

**Comment by enki-code** [comment]
I noticed this the other day and completely forgot about it. We'll look into this and incorporating the related feature request (if possible).

**Comment by catb0t** [comment]
@enki-code Just think, if you don't fix this, SOX's tagline can be [Kinda like Lemmings](http://minecraft.wikia.com/wiki/Splash_Texts)!

**Comment by shu8** [comment]
<img width="317" alt="screen shot 2016-04-06 at 09 46 14" src="https://cloud.githubusercontent.com/assets/8850830/14310911/6e1b04d0-fbdc-11e5-9a89-e6cbad0c32ec.png">

So do you not see this? @catb0t

**Comment by catb0t** [comment]
@shu8 Wat.

![wat](https://cloud.githubusercontent.com/assets/3238748/14318048/1b4bf9ba-fbd9-11e5-870e-c5d7f1f25101.png)

**Comment by shu8** [comment]
@catb0t hmmm that's *very* weird.. @enki-code can you confirm you also see that?

I can't see why this is happening... are you up to date? Try installing the *master* version -- v1.0.3 has been pushed there now...

**Comment by enki-code** [comment]
@shu8 
**v.1.0.3 official**

![pointer](https://cloud.githubusercontent.com/assets/12533449/14321838/a61fbfa4-fbcf-11e5-9fda-cba5aa2132da.png)

**Comment by shu8** [comment]
@enki-code are you free for chat?

**Comment by shu8** [comment]
@catb0t @enki-code I've pushed a fix to DEV. please could you reinstall the dev version and confirm?

**Comment by enki-code** [comment]
@shu8 confirmed

**Comment by enki-code** [comment]
@catb0t @shu8 

What do you think of using the chevrons instead of those arrows?

![linkedtofrom](https://cloud.githubusercontent.com/assets/12533449/14511102/0a6d18a2-018a-11e6-8070-ae875fdcce27.png)

And using BOTH when linking occurs in both directions?

**Comment by catb0t** [comment]
@enki-code Nice, I like!
